apache-rat-core/src/main/java/org/apache/rat/Report.java

```diff
@@ -147,27 +146,106 @@ public static final void main(String[] args) throws Exception {
         if (args == null || args.length != 1) {
             printUsage(opts);
         } else {
+            
```

Comment:
Is the commented our code what you mean in RAT-323 to need harmonizing?

Reply:
I'll merge the changes, feel free to remove the commentedOut stuff in a different PR - thanks.